mteb/tasks/STS/pl/PolishSTS.py

```diff
@@ -15,7 +15,7 @@ class SickrPLSTS(AbsTaskSTS):
         eval_splits=["test"],
         eval_langs=["pl"],
         main_score="cosine_spearman",
-        revision="a6ea5a8cab320b040a23452cc28066d9beae2cee",
+        revision="fd5c2441b7eeff8676768036142af4cfa42c1339",
```

Comment:
Why are the revision values changed here?

Reply:
Most of the changes was due to the test stating that the revision was not found.